src/Chainweb/Logging/Config.hs

```diff
@@ -90,7 +90,7 @@ defaultLogConfig = LogConfig
     , _logConfigBackend = defaultBackendConfig
     , _logConfigTelemetryBackend = defaultEnableConfig defaultBackendConfig
     , _logConfigAmberdataBackend = EnableConfig False defaultAmberdataConfig
-      -- ^ Amberdata logging disabled by default
+        -- Amberdata logging disabled by default
```

Comment:
I noticed this elsewhere too. I'm surprised that CI didn't catch it. Perhaps something to do with specific Haddock versions on CI vs our machines.